client/src/pages/ThesisPage/components/ThesisCommentsForm/ThesisCommentsForm.tsx

```diff
@@ -0,0 +1,52 @@
+import { Button, Stack, Textarea } from '@mantine/core'
+import { useState } from 'react'
+import UploadFileModal from '../../../../components/UploadFileModal/UploadFileModal'
+import { useThesisCommentsContext } from '../../../../contexts/ThesisCommentsProvider/hooks'
+import { Upload } from 'phosphor-react'
+
+const ThesisCommentsForm = () => {
+  const { postComment, posting } = useThesisCommentsContext()
+
+  const [message, setMessage] = useState('')
+  const [file, setFile] = useState<File>()
+
+  const [uploadModal, setUploadModal] = useState(false)
+
+  return (
+    <Stack>
+      <UploadFileModal
+        title='Attach File'
+        opened={uploadModal}
+        onClose={() => setUploadModal(false)}
+        onUpload={setFile}
+      />
+      <Textarea
+        placeholder='Add a comment'
```

Comment:
The placeholder disappears once the user starts typing.
You might consider introducing a label here to improve UX.